src/5.0_to_5.1/migrateSavedWidget.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  AWidgetState as AWidgetState50,
+  deserializeWidgetState,
+} from "@activeviam/activeui-sdk-5.0";
+import {
+  AWidgetState as AWidgetState51,
+  serializeWidgetState,
+} from "@activeviam/activeui-sdk-5.1";
+import { MigrateWidgetCallback } from "../migration.types";
+import { produce } from "immer";
+import { mutateWidget } from "./mutateWidget";
+
+/**
+ * Returns the 5.1 migrated version of the 5.0 `serializedWidgetState`.
+ * Used only for saved widgets, not for widgets within dashboards.
+ */
```

Comment:
If we go for `serialize`/`deserialize` at a higher level in `getMigrateXyz`, then this comment and the implementation with the serialized state doesn't work anymore.
Same for the other `migrateXyz` functions.